test/behave/skill-alarm.feature

```diff
@@ -122,21 +117,20 @@ Feature: Alarm skill functionality
 
   Examples: user sets an alarm without specifiying am or pm
     | set an alarm for a time without am or pm |
-    | alarm for 12 |
+    | set alarm for 12 |
 
-  @xfail
   # Jira MS-67 https://mycroft.atlassian.net/browse/MS-67
-  Scenario Outline: Failing set an alarm for a duration instead of a time
+  Scenario Outline: set an alarm for a duration instead of a time
     Given an english speaking user
      And there are no previous alarms set
      When the user says "<set an alarm for a duration>"
      Then "mycroft-alarm" should reply with dialog from "alarm.scheduled.for.time.dialog"
 
   Examples:
     | set an alarm for a duration |
-    | set an alarm for 30 minutes |
-    | alarm in 5 minutes |
-    | alarm in 5 minutes and 30 seconds |
+    | create an alarm for 30 minutes |
+    | set alarm in 5 minutes |
+    | new alarm in 5 minutes and 30 seconds |
```

Comment:
For utterances without a verb, I think we need to either:
- create an alarm
- catch the utterance and disambiguate

Currently the Alarm Skill won't trigger from this utterance and so you would either get the Date Time Skill or a fallback even though we have the keyword alarm and a time / duration.

My assumption is that this should be a Padatious intent so it only matches specific short utterances and we don't have any utterance with the word "alarm" triggering this Skill.
eg 
```
alarm
alarm in {duration}
alarm at {time}
```

Reply:
So, where to start. First, these tests all pass so the behavior Derick requested was achieved. The utterance 'alarm 6' will now create an alarm at 6. Second, indeed, this triggers for nearly anything which contains the word 'alarm' in it. I don't know if that's a good or bad thing as we are reaching the point of 'the little boy and the dyke' mode where you catch something new here and lose something old there. This will get worse before it gets better with our current design which is why I push for new requirements to be specified using VK tests. This will be the only hope we have of catching intent conundrums as we move forward. How that squares with reducing test coverage by removing tests to decrease test execution times is left as an exercise for the reader.